torch/csrc/jit/ir.cpp

```diff
@@ -777,8 +777,8 @@ bool Node::matches(
 bool Node::mustBeNone() const {
   return kind_ == prim::AutogradZero ||
       (kind_ == prim::Constant && !this->hasAttributes() &&
-       (output()->type()->cast<OptionalType>() ||
-        output()->type() == NoneType::get()));
+       output()->type()->cast<OptionalType>()) ||
```

Comment:
Good fix!

Reply:
:slightly_smiling_face:

Reply:
Actually, very bad breakage! It throws if we have several outputs. That case needs to be covered in `Value::mustBeNone`.